tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "isolatedModules": false,
     "jsx": "react",
     "noImplicitAny": false,
+    "noImplicitThis": false,
```

Comment:
Please take this out and fix the issue (you may have to explicitly type `this`)

Reply:
I'm not sure how you can fix this, the error it complains about is in a text block within a file

Reply:
See comment above :)